src/vs/workbench/api/common/extHostNotebookKernels.ts

```diff
@@ -387,6 +399,12 @@ export class ExtHostNotebookKernels implements ExtHostNotebookKernelsShape {
 				}
 			}
 		}
+
+		// If we're interrupting all cells, we also need to cancel the notebook level execution.
+		const items = this._activeNotebookExecutions.get(document.uri);
```

Comment:
Hm, does this API make sense for kernels that don't use Interrupt? Maybe it should only be usable for those kernels.

Reply:
Agreed